mktestdocs/__init__.py

```diff
@@ -1,4 +1,5 @@
 from mktestdocs.__main__ import (
+    register_executor,
```

Comment:
Making this public seems sensible, as it allows other languages to be added more easily in custom pipelines. But I wonder now if this should be documented too. Do you happen to have a nice example for it ready? If not it's no disaster.

Reply:
The registry was always intended to allow future languages to be supported without the need for more pull requess. I don't have an example ready other than the bash addition, but I can add documentation around this new API.

Reply:
Yes please! I can imagine that a small description on the readme is also useful to refer to in the error message.